Doc/library/constants.rst

```diff
@@ -65,8 +65,9 @@ A small number of constants live in the built-in namespace.  They are:
 .. index:: single: ...; ellipsis literal
 .. data:: Ellipsis
 
-   The same as the ellipsis literal "``...``". Special value used mostly in conjunction
-   with extended slicing syntax for user-defined container data types.
+   The same as the ellipsis literal "``...``", an object frequently used as a
+   placeholder for another value. Assignment to  ``Ellipsis`` is possible, but
```

Comment:
I'm not sure "a placeholder for another value" is a good summary. I don't think it describes many of the typing use cases, for one.

Maybe "to indicate that something is omitted"?